<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUserWindow</class>
 <widget class="QMainWindow" name="EditUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>用户名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>密码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>设置新密码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>300</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSubmit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonCancel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>取消修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>真实姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditRealName">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>未认证</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>证件号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIDNumber">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>未认证</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>手机号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPhone">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>未认证</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>如需修改认证信息，请重新提交认证</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSellerApply">
            <property name="text">
             <string>前往认证</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>默认地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDefaultAddress">
        <property name="text">
         <string>未设置默认地址</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonNewAddress">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetAddress">
      <column>
       <property name="text">
        <string>地址</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>收件人</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>电话</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>删除</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>默认</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
